action.yml

```diff
@@ -80,6 +83,7 @@ runs:
         fetch-depth: '${{ inputs.checkout-fetch-depth }}'
         path: '${{ inputs.checkout-path }}'
         persist-credentials: '${{ inputs.checkout-persist-credentials }}'
+        ref: '${{ inputs.checkout-ref }}'
```

Comment:
I'm assuming it was tested that this continues to work if checkout-ref is not passed in?

Reply:
Yes, this was tested.

Reply:
Additionally this was tested on the default branch, selecting a different branch (which was the intended use case of this PR), and with selecting a branch did not exist. All cases performed how I would expect.